src/Git/FetchAndSetCurrentUserByReplacingCurrentOriginRemote.php

```diff
@@ -21,19 +21,14 @@ public function __invoke(
         UriInterface $repositoryUri,
         string $repositoryRootDirectory
     ): void {
-        (new Process(['git', 'remote', 'rm', 'origin'], $repositoryRootDirectory))
-            ->run();
-
-        (new Process(['git', 'remote', 'add', 'origin', $repositoryUri->__toString()], $repositoryRootDirectory))
-            ->mustRun();
-
-        (new Process(['git', 'fetch', 'origin'], $repositoryRootDirectory))
-            ->mustRun();
-
-        (new Process(['git', 'config', 'user.email', $this->variables->gitAuthorEmail()], $repositoryRootDirectory))
-            ->mustRun();
-
-        (new Process(['git', 'config', 'user.name', $this->variables->gitAuthorName()], $repositoryRootDirectory))
-            ->mustRun();
+        try {
+            Shell\execute('git', ['remote', 'rm', 'origin'], $repositoryRootDirectory);
+        } catch (Shell\Exception\FailedExecutionException) {
```

Comment:
Possibly to be tested (mutant should occur here)

Reply:
done 👍